### DISCUSSION TITLE
Request models list from LocalAI

### DISCUSSION THREAD

**Discussion by mounta11n** [lisiur/ChatWizard#53] [Ideas]
Hi, thank you for your great app!

I wanted to ask for a feature: Could you please implement a possibility to request and refresh the models list by `curl http://localhost:8080/v1/models`. Since it is really cool that one can easily change the api endpoint/url, i was able to make requests with localAI, but I have to rename my local llama models either gpt-3.5 or gpt-4 and put them in LocalAI's models folder, and therefore I can only use two different models.

It would be extremly nice, if one could edit the models list/add more items to it or if you could just refresh it by the above curl request.

I tried to modify the source code myself, but I am absolutely not familiar with rust and therefore I greatly failed with this attempt.

Really hope you could make this changes, because i think your app has great potential for the local-llm community 🙏🏾

**Comment by lisiur** [comment]
Thank you for your affirmation. I will address this issue when I have the opportunity.  See #54
